recipe/meta.yaml

```
@@ -55,11 +54,13 @@ requirements:
     - typing-extensions >=4.1.0,<5
     - xgboost >=1.7.3,<2
     - cachetools >=3.1.1,<5
```

```suggestion
    - cachetools >=3.1.1,<6
```
See https://inspector.pypi.io/project/snowflake-ml-python/1.2.3/packages/52/54/7ae7bc8c18358bd5dd9cadcd325da8ad0e3627a7e94bb6d255661beab0b5/snowflake_ml_python-1.2.3-py3-none-any.whl/snowflake_ml_python-1.2.3.dist-info/METADATA#line.238